src/main/protobuf/soc/inventory.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+
+package protos.soc;
+
+enum Resource {
+    UNKNOWN_R = 0;
+    BRICK = 1;
+    ORE = 2;
+    SHEEP = 3;
+    WHEAT = 4;
+    WOOD = 5;
+}
+
+enum DevelopmentCard {
+    UNKNOWN_D = 0;
```

Comment:
I don't like having `UKNOWN_D` as one of the development cards. It's not a development card. Same for `UNKNOWN_R`

Reply:
Also, while we're talking about enums, I started all my enums with a `NONE` value. This is because if an enum is not required in a message, even if its not supplied, instead of defaulting to `null` it will default to `0`, which will be the first index of that enum. 

IN cases (like `Hex.Resource`) where th enum is nested/must always be defined on its message type, this isn't really an issue. For all non-nested enums though, I think we should always define a `null` default value. even for the nested ones it's tempting in case the design pattern changes. If we don't have it, we've really locked ourselves in.

Reply:
I wanted to name them both just `UNKNOWN ` but the compiler did not like that.

Reply:
But is `UNKNOWN` the same as `NONE` ?

Reply:
This idea was more for if we are hiding card knowledge. Instead of stealing an Ore your stealing an Unknown Card